### PR TITLE
[os_net_setup] Update public allocation pool and network configuration

### PR DESCRIPTION
**Pull request by fyanac**
Extend public allocation pool

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running

### CODE REVIEW

File: roles/os_net_setup/defaults/main.yml [hunk 1]
```diff
@@ -6,15 +6,17 @@ cifmw_os_net_setup_osp_calls_delay: 5
 cifmw_os_net_setup_config:
   - name: public
     external: true
+    shared: true
+    mtu: 1330
```
Comment (karelyatin) [inline roles/os_net_setup/defaults/main.yml:10]: why mtu reduction?

Apart from that all looks good.

**Review by karelyatin — CHANGES_REQUESTED** [review]

File: roles/os_net_setup/defaults/main.yml (hunk 1 above)
Comment (rsafrono) [reply roles/os_net_setup/defaults/main.yml:10]: This is for whitebox path mtu discovery tests (the ones under test_mtu.py). For this coverage our tripleo jobs had external network mtu 1500 and tenant mtu 9000. When we were planning osp18 jobs I suggested for simplification to use normal tenant networks mtu but just reduce external network mtu. So this is it.
Maybe we should have set this only for neutron jobs, not for everybody.

Comment (karelyatin) [reply roles/os_net_setup/defaults/main.yml:10]: +1 to only override where it's really needed, not in default

Comment (fyanac) [reply roles/os_net_setup/defaults/main.yml]: I removed mtu reduction

**Review by rsafrono — APPROVED** [review]
lgtm

**Review by karelyatin — APPROVED** [review]

**Pull request merged**